File: Solarness.Services/Database/Notification.cs
using System;
using System.Collections.Generic;

namespace Solarness.Services.Database;

public partial class Notification
{
    public int NotificationId { get; set; }

    public int? ProjectId { get; set; }

    public string? Content { get; set; }

    public string? Title { get; set; }

    public DateTime? SendDate { get; set; }

    public virtual Project? Project { get; set; }
}

File: UI/solarness_mobile/lib/models/Notification/notification.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:solarness_mobile/models/Project/project.dart';

part 'notification.g.dart';

@JsonSerializable()
class Notifications {
  int? id;
  int? projectId;
  String? title;
  String? content;
  DateTime? sendDate;
  Project? project;

  Notifications( this.id, this.projectId, this.title,this.content, this.sendDate,this.project);

  factory Notifications.fromJson(Map<String, dynamic> json) => _$NotificationsFromJson(json);

  Map<String, dynamic> toJson() => _$NotificationsToJson(this);
}
